examples/javaee/README.md

```diff
@@ -0,0 +1,158 @@
+
+The order of Service and the targeted Pods does not matter. However Service needs to be started before any other Pods consuming the Service are started.
+
+In this application, we will use a Kubernetes Service to provide a discoverable endpoints for the MySQL endpoint in the cluster.  MySQL service target pods with the labels `name: mysql-pod` and `context: docker-k8s-lab`.
+
+Here is definition of the MySQL service: [mysql-service.yaml](mysql-service.yaml)
+
+Create this service:
+
+```sh
+kubectl create -f examples/javaee/mysql-service.yaml
+```
+
+Get status of the service:
+
+```sh
+kubectl.sh get -w se
+NAME            LABELS                                    SELECTOR                                IP(S)          PORT(S)
+kubernetes      component=apiserver,provider=kubernetes   <none>                                  10.247.0.1     443/TCP
+mysql-service   context=docker-k8s-lab,name=mysql-pod     context=docker-k8s-lab,name=mysql-pod   10.247.63.43   3306/TCP
+```
+
+If multiple services are running, then it can be narrowed by specifying labels:
+
+```sh
+kubectl.sh  get -w po -l context=docker-k8s-lab,name=mysql-pod
+NAME        READY     STATUS    RESTARTS   AGE
+mysql-pod   1/1       Running   0          4m
+```
+
+This is also the selector label used by service to target pods.
+
+When a Service is run on a node, the kubelet adds a set of environment variables for each active Service. It supports both Docker links compatible variables and simpler `{SVCNAME}_SERVICE_HOST` and `{SVCNAME}_SERVICE_PORT` variables, where the Service name is upper-cased and dashes are converted to underscores.
+
+Our service name is ``mysql-service'' and so MYSQL_SERVICE_SERVICE_HOST and MYSQL_SERVICE_SERVICE_PORT variables are available to other pods. This host and port variables are then used to create the JDBC resource in WildFly.
+
+### Start WildFly Replication Controller
+
+WildFly is a lightweight Java EE 7 compliant application server. It is wrapped in a Replication Controller and used as the Java EE runtime.
+
+In Kubernetes a [_Replication Controller_](../../docs/user-guide/replication-controller.md) is responsible for replicating sets of identical pods. Like a _Service_ it has a selector query which identifies the members of it's set.  Unlike a service it also has a desired number of replicas, and it will create or delete pods to ensure that the number of pods matches up with it's desired state.
+
+Here is definition of the MySQL service: [wildfly-rc.yaml](wildfly-rc.yaml). Create this controller:
+
+```sh
+kubectl create -f examples/javaee/wildfly-rc.yaml
+```
+
+Check status of the pod inside replication controller:
+
+```sh
+kubectl.sh get po
+NAME               READY     STATUS    RESTARTS   AGE
+mysql-pod          1/1       Running   0          1h
+wildfly-rc-w2kk5   1/1       Running   0          6m
+```
+
+### Access the application
```

Comment:
This section should use the apiserver proxy so the user can access the app in his/her browser. It should also link to https://github.com/GoogleCloudPlatform/kubernetes/blob/master/docs/user-guide/connecting-to-applications-proxy.md